packages/editor/src/client/blocks/richTextBlock.tsx

```diff
@@ -101,7 +101,11 @@ const ElementTags: any = {
   UL: () => ({type: BlockFormat.UnorderedList}),
   TB: () => ({type: BlockFormat.Table}),
   TR: () => ({type: BlockFormat.TableRow}),
-  TD: () => ({type: BlockFormat.TableCell})
+  // TD: () => ({type: BlockFormat.TableCell}),
+  TD: (el: Element) => ({
```

Comment:
Try using a more meaningful name for `el`.

Reply:
`el` is a known abbreviation